### PR TITLE
Added first implementation

### PR DESCRIPTION
**Pull request by sd-lueckel**

### CODE REVIEW

File: tests/test.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - solutiondrive.grafana
```
Comment (michz) [inline tests/test.yml]: Newline

**Review by michz — CHANGES_REQUESTED** [review]

**Review by michz — DISMISSED** [review]

**Review by Jnoack331 — DISMISSED** [review]

**Comment by michz** [comment]
Same permission problem here @solutionDrive/maintainer

**Comment by sd-lueckel** [comment]
permissions should be fine now @solutionDrive/maintainer

**Comment by sd-lueckel** [comment]
Ready for review again @solutionDrive/maintainer

**Review by michz — DISMISSED** [review]

**Review by michz — APPROVED** [review]
🎉

**Review by Jnoack331 — APPROVED** [review]
👍

**Review by mattagohni — APPROVED** [review]

**Pull request merged**